### PR TITLE
FLOW-125: UI dropdown menu for verification types and FLOW-162: Phone Call Verification

### PR DESCRIPTION
**Pull request by tle-docusign**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/DocuSignDemo/apiDefinition.swagger.json [hunk 1]
```diff
@@ -630,11 +630,32 @@
...
+            "x-ms-summary": "Verification Type",
+            "x-ms-dynamic-values": {
+              "operationId": "StaticResponseForVerificationTypes",
+              "value-collection": "verificationTypes",
+              "value-path": "type",
+              "value-title": "name"
+            }
+          },
+          {
+            "name":"routingOrder",
```
Comment (angelga-docusign) [inline certified-connectors/DocuSignDemo/apiDefinition.swagger.json:644]: @tle-docusign we still have this issue. You promoted routingorder to always show up in the UI. It shoudl be under advanced options. Just remove it from here, and restore it to under "AdditionalRecipientData".

File: certified-connectors/DocuSignDemo/script.csx [hunk 2]
```diff
@@ -84,6 +84,31 @@ public class Script : ScriptBase
...
 
+    if (operationId.Equals("StaticResponseForVerificationTypes", StringComparison.OrdinalIgnoreCase))
+    {
+      var verificationTypesArray = new JArray();
+
+      string [,] verificationTypes = {
+        {"accessCode", "Access Code"},
+        {"phoneCall", "Phone Call"},
+        {"SMS", "SMS"},
+        {"none", "None"},
```
Comment (angelga-docusign) [inline certified-connectors/DocuSignDemo/script.csx]: How do we make sure that None is selected by default?

File: certified-connectors/DocuSignDemo/script.csx [hunk 3]
```diff
@@ -485,23 +538,43 @@ public class Script : ScriptBase
...
+  private void AddCoreRecipientParams(JArray signers, JObject body) 
+  {
     var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
-    signers[0]["name"] = Uri.UnescapeDataString(query.Get("recipientName")).Replace("+", " ");
-    signers[0]["email"] = Uri.UnescapeDataString(query.Get("recipientEmail")).Replace("+", " ");
-    if (string.IsNullOrWhiteSpace((string)signers[0]["recipientId"]))
+    signers[0]["recipientId"] = Guid.NewGuid();
+    if (!string.IsNullOrEmpty(query.Get("routingOrder")))
     {
-      signers[0]["recipientId"] = Guid.NewGuid();
```
Comment (angelga-docusign) [inline certified-connectors/DocuSignDemo/script.csx:496]: Not sure why you're removing this

**Review by angelga-docusign — CHANGES_REQUESTED** [review]
Please fix the signing order comment. Then we can review the rest together.

**Review by angelga-docusign — APPROVED** [review]
Let's fix the Node by default setting

File: certified-connectors/DocuSignDemo/script.csx (hunk 3 above)
Comment (tle-docusign) [reply certified-connectors/DocuSignDemo/script.csx:496]: I just relocated it to line 532

File: certified-connectors/DocuSignDemo/apiDefinition.swagger.json [hunk 4]
```diff
@@ -630,11 +630,28 @@
...
           },
+          {
+            "name": "verificationType",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Verification Type",
+            "x-ms-summary": "Verification Type",
+            "enum":["None", "Access Code", "ID Verification", "Phone Call", "Knowledge-Based","SMS"],
+            "default": "None"
```
Comment (angelga-docusign) [inline certified-connectors/DocuSignDemo/apiDefinition.swagger.json:641]: Nice!

**Review by angelga-docusign — APPROVED** [review]

**Pull request merged**